### PR TITLE
feat: add arbitrum goerli

### PR DESCRIPTION
**Pull request by gzeoneth**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

This PR add support for Arbitrum Goerli Testnet (chainId: 421613)

- **Other information**:

The Arbitrum Team have deployed an instance of Uniswap V3 on Arbitrum Goerli as follow
```
const ARBITRUM_GOERLI_V3_CORE_FACTORY_ADDRESSES = '0x4893376342d5D7b3e31d4184c08b265e5aB2A3f6'
const ARBITRUM_GOERLI_ROUTER_ADDRESS = '0xab7664500b19a7a2362Ab26081e6DfB971B6F1B0'
const ARBITRUM_GOERLI_V3_MIGRATOR_ADDRESSES = '0xA815919D2584Ac3F76ea9CB62E6Fd40a43BCe0C3'
const ARBITRUM_GOERLI_MULTICALL_ADDRESS = '0x8260CB40247290317a4c062F3542622367F206Ee'
const ARBITRUM_GOERLI_QUOTER_ADDRESSES = '0x1dd92b83591781D0C6d98d07391eea4b9a6008FA'
const ARBITRUM_GOERLI_NONFUNGIBLE_POSITION_MANAGER_ADDRESSES = '0x622e4726a167799826d1E1D150b076A7725f5D81'
const ARBITRUM_GOERLI_TICK_LENS_ADDRESSES = '0xb52429333da969a0C79a60930a4Bf0020E5D1DE8'
```

Dependent on https://github.com/Uniswap/smart-order-router/pull/188

### CODE REVIEW

File: src/components/DowntimeWarning/index.tsx
Comment (vm) [inline src/components/DowntimeWarning/index.tsx:67]: this was deleted, so you can rebase against main to update

File: src/constants/tokens.ts
Comment (vm) [inline src/constants/tokens.ts:77]: after you rebase you may need to remove the export on this const

**Review by vm — APPROVED** [review]
after conflicts are fixed this LGTM

**Comment by vm** [comment]
will merge after the smart order router change is merged

**Comment by gzeoneth** [comment]
> will merge after the smart order router change is merged

ready to merge

**Comment by gzeoneth** [comment]
Need https://github.com/Uniswap/sdk-core/pull/36 to fix the optional cypress test

File: yarn.lock
Comment (zzmp) [inline yarn.lock:5146]: This must coalesce with 1.2.1.

File: yarn.lock
Comment (zzmp) [inline yarn.lock:5032]: This must coalesce with the entry above it.

File: yarn.lock
Comment (zzmp) [inline yarn.lock:5044]: This should go away.

**Review by zzmp — CHANGES_REQUESTED** [review]

**Review by zzmp — CHANGES_REQUESTED** [review]
manifest/lockfile changes should be made in a separate PR to make them easier to review and reduce the chance of dependency version skew (eg this introduces a version mismatch with sdk-core).

This PR should wait until https://github.com/Uniswap/interface/pull/5806 is merged, and then merge that in to take care of the package/lockfile updates.

File: patches/@uniswap+smart-order-router+2.10.0.patch
Comment (vm) [inline patches/@uniswap+smart-order-router+2.10.0.patch:35]: what is this line doing?

Comment (gzeoneth) [reply patches/@uniswap+smart-order-router+2.10.0.patch]: This add the ArbGoerli version of USDC, the comment is incorrect and removed

**Review by vm — APPROVED** [review]

**Pull request merged**
